azkaban-web-server/src/test/java/azkaban/imagemgmt/utils/JsonUtils.java

```diff
@@ -0,0 +1,20 @@
+package azkaban.imagemgmt.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.apache.commons.io.IOUtils;
+
+public class JsonUtils {
```

Comment:
There is a class called JSONUtils in az-core module. Can you please see if that can be used and move readJsonAsString if it already doesn't exist over there?

Reply:
Done.